internal/hud/server/server.go

```diff
@@ -64,6 +64,7 @@ func ProvideHeadsUpServer(store *store.Store, assetServer assets.Server, analyti
 	r.HandleFunc("/api/sail", s.HandleSail)
 	r.HandleFunc("/api/trigger", s.HandleTrigger)
 	r.HandleFunc("/api/snapshot/new", s.HandleNewSnapshot)
+	r.HandleFunc("/api/snapshot/{snapshot_id}", s.SnapshotJSON)
```

Comment:
is `{snapshot_id}` actually used here, is it only to match the path used by the snapshot server?

is this endpoint only for local testing? seems worth a comment

Reply:
Yes and yes, will add a comment.